diracx-cli/src/diracx/cli/internal/legacy.py

```diff
@@ -364,6 +364,13 @@ def generate_helm_values(
     diracx_settings["DIRACX_SANDBOX_STORE_BUCKET_NAME"] = (
         f"{diracx_hostname.split('.')[0]}-sandboxes"
     )
+    try:
+        diracx_settings["DIRACX_SANDBOX_STORE_SE_NAME"] = cfg["Systems"][
+            "WorkloadManagement"
+        ]["Services"]["SandboxStore"]["LocalSE"]
```

Comment:
```suggestion
        diracx_settings["DIRACX_SANDBOX_STORE_SE_NAME"] = cfg["Systems"][
            "WorkloadManagement"
        ]["Services"]["SandboxStore"].get("LocalSE", "SandboxSE")
```

Reply:
That's not needed, as it already is the default in the pydantic model